lte/gateway/c/core/oai/tasks/amf/amf_app_main.cpp

```diff
@@ -58,16 +58,19 @@ static int handle_message(zloop_t* loop, zsock_t* reader, void* arg) {
   amf_app_desc_t* amf_app_desc_p = get_amf_nas_state(false);
   imsi64_t imsi64                = itti_get_associated_imsi(received_message_p);
 
+  OAILOG_INFO(
```

Comment:
should this be DEBUG level?

Reply:
AMF task received and sent out messages defined INFO level.
I mean interaction messages between AMF--NGAP, AMF--SubscriberDB, AMF--SMF.

please have a look AMF_INFO log messages  :

![image](https://user-images.githubusercontent.com/83060027/128997436-06984042-b3ab-459a-a84c-ea65a9bc800b.png)

Suggest me if improvement is needed?